app/Http/Controllers/OrganizationController.php

```diff
@@ -19,6 +19,27 @@
 class OrganizationController extends Controller
 {
 
+    /**
+     * Disable an organization
+     *
+     * @param Request $request
+     * @param integer $orgId
+     * @return JsonResponse
+     */
+    public function disable(Request $request, int $orgId): JsonResponse 
```

Comment:
are we going to use the 'resources' style for public controller methods: index,show,update,etc?